### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by dierbei**
Update README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 ![](docs/images/logo.png)
 
+<p align="center">
+  <a href="https://github.com/mxpv/shim-rs/actions/workflows/ci.yml">
+    <img alt="GitHub Workflow Status" src="https://github.com/mxpv/shim-rs/actions/workflows/ci.yml/badge.svg?branch=main">
+  </a>
```
Comment (kevin-wangzefeng) [inline README.md:6]: Should fetch CI status from kuasar-io/kuasar actions

```suggestion
  <a href="https://github.com/kuasar-io/kuasar/actions/workflows/ci.yml">
    <img alt="GitHub Workflow Status" src="https://github.com/kuasar-io/kuasar/actions/workflows/ci.yml/badge.svg?branch=main">
  </a>
```

**Review by kevin-wangzefeng — COMMENTED** [review]
minor comments, thanks

File: README.md (hunk 1 above)
Comment (dierbei) [reply README.md:6]: @kevin-wangzefeng Thanks for the suggestion, my carelessness made this mistake. Feel sorry.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
 ![](docs/images/logo.png)
 
+<p align="center">
+  <a href="https://github.com/kuasar-io/kuasar/actions/workflows/ci.yml">
+    <img alt="GitHub Workflow Status" src="https://github.com/kuasar-io/kuasar/actions/workflows/ci.yml/badge.svg?branch=main">
+  </a>
+  <a href="https://app.slack.com/client/T08PSQ7BQ/C052JRURD8V">
```
Comment (Burning1020) [inline README.md:7]: Please update the latest link from README.md.

Comment (dierbei) [reply README.md]: Now updated with the latest slack link.

**Pull request merged**